solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -750,12 +750,14 @@ public void load() {
 
     zkSys.initZooKeeper(this, cfg.getCloudConfig());
     if (isZooKeeperAware()) {
+      log.info("initializing PKIAuth");
       pkiAuthenticationPlugin = new PKIAuthenticationPlugin(this, zkSys.getZkController().getNodeName(),
           (PublicKeyHandler) containerHandlers.get(PublicKeyHandler.PATH));
       // use deprecated API for back-compat, remove in 9.0
       pkiAuthenticationPlugin.initializeMetrics(
           solrMetricsContext.metricManager, solrMetricsContext.registry, solrMetricsContext.tag, "/authentication/pki");
       TracerConfigurator.loadTracer(loader, cfg.getTracerConfiguratorPluginInfo(), getZkController().getZkStateReader());
```

Comment:
Just to be exhaustive : Can we add `initializing Tracer` here as well? (so we can tell the timing between PKIAuth and trace) :)

Reply:
@patsonluk please update the PR appropriate. We want to deploy this on staging today. Thanks!

Reply:
Added in https://github.com/fullstorydev/lucene-solr/pull/132/commits/c21c4c182396f25496c16d97eacba647ba394764